rfcs/convergence/fixed-versions-for-prerelease.md

```diff
@@ -154,7 +154,22 @@ To mitigate this problem during the prerelease phase we can do the following:
 ### Prefer to release new components straight into react-components beta
 
 For sane versioning, new packages should be released straight into the `beta` prerelease tag. A new way of consuming
-unstable packages is proposed in the next section.
+unstable packages is proposed in the next section. `alpha` and `beta` are liable to breaking API changes both internally
+and externally. Having both prerelease tags existing side by side could cause dependency hell too easily.
+
+### RC release can coexist with beta components
+
+As Fluent v9 moves to the `RC` stage, we want to clearly communicate to partners what they can expect from the stable
+release of Fluent. It would not make sense to elevate `beta` components to `rc` release if we do not expect to release
+those components in the first stable release of Fluent v9.
+
+Since RC will solidify our internal APIs and concepts in our core utilities, we **do not plan to make breaking changes**.
+While the previous problems with prerelease tags still exist, the RC status will be more stable than any `alpha` or `beta`
```

Comment:
does this mean following workflow?

components that are part of `unstable` api in react-components are basically still in beta thus the need of bumping beta version/release tag?

Reply:
That there will be no more `ustable` api necessary for `rc` and `beta`, since we will have the two prerelease tags existing side by side

Reply:
gotcha, thx!